Readme_Direct_Ethernet.txt

```diff
@@ -0,0 +1,11 @@
+The direct ethernet is no longer being actively developed here. To
+avoid confusion it has been removed from master.
+
+The latest binary release (1.2.3) is available via bintray:
+
+https://bintray.com/labrad/generic/direct-ethernet
+
+The source code is available in a release tag of the git repository:
+
+https://github.com/martinisgroup/servers/tree/direct-ethernet-v1.2.3
```

Comment:
I would suggest putting all these notes in the main `README.md` so this is shows up for people browsing the repo in the web interface.